Tests/Unit Tests/Glass.Mapper.Sc.FakeDb/ScVersionTest.cs

```diff
@@ -81,6 +81,12 @@ public void Sc103()
         {
             Assert.True(true);
         }
+#endif
+#if SC103

```

Comment:
```suggestion
#if SC104

```
(also wrong on rule 79, should be `#if SC103` there.